.travis.yml

```diff
@@ -196,9 +196,29 @@ jobs:
       script:
       - python setup.py build_ext --inplace
       - pip install "qiskit-aer"
-      - travis_wait 45 make test_randomized
+      - make test_randomized
+
+    - stage: test
+      name: Benchmarks
+      <<: *stage_linux
+      script:
+      - python setup.py build_ext --inplace
+      - pip install "qiskit-aer"
+      - pip install "asv"
+      - git clone "https://www.github.com/Qiskit/qiskit" meta-qiskit
+      - >-
+        sed -i
+        -e 's/"repo": "https:\/\/github.com\/Qiskit\/qiskit-terra.git",/"repo": "..",/'
+        -e 's/"pythons": \["3.5", "3.6", "3.7"\],/"pythons": ["3.5"],/'
```

Comment:
We can just specify this on the asv command with `-E pythons:3.5` (or something like that) instead of editing the config file.